src/Shouldly/Configuration/DiffTools/VimDiff.cs

```diff
@@ -0,0 +1,17 @@
+﻿#if ShouldMatchApproved
+namespace Shouldly.Configuration.DiffTools
+{
+    internal sealed class VimDiff : DiffTool
+    {
+        public VimDiff() : base("VimDiff", new DiffToolConfig
+        {
+            WindowsPath = @"vim.bat",
```

Comment:
is this really safe to assume on all machines?

Reply:
Typical [vim for windows installer](https://github.com/vim/vim-win32-installer/releases/tag/v8.2.1145) puts several *.bat files into `%WINDIR%` folder when option `Create .bat files` selected. Search for editor in %PATH% seemed more reasonable to me. What do you think?